packages/ra-ui-materialui/src/input/NullableBooleanInput.spec.tsx

```diff
@@ -26,24 +26,24 @@ describe('<NullableBooleanInput />', () => {
             />
         );
         const select = getByRole('button');
-        fireEvent.click(select);
+        fireEvent.mouseDown(select);
```

Comment:
Why ? Comment maybe ?

Reply:
That's the way the material-ui team tests their Select component. I didn't looked further when the react-admin Select-based tests stopped passing after upgrading material-ui to the latest version.